### PR TITLE
[HOPSWORKS-944] kerberos changes

### PR DESCRIPTION
**Pull request by ErmiasG**
## Make sure there is no duplicate PR for this issue

* **Please check if the PR meets the following requirements**
- [ ] Adds tests for the submitted changes (for bug fixes & features)
- [x] Passes the tests
- [x] HOPSWORKS JIRA issue has been opened for this PR
- [x] All commits have been squashed down to a single commit

* **Post a link to the associated JIRA issue**

* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)

* **What is the new behavior (if this is a feature change)?**

* **Does this PR introduce a breaking change?** (What changes might users need to make in their application due to this PR?)

* **Other information**:

### CODE REVIEW

**Comment by ErmiasG** [comment]
logicalclocks/hops-testing#297

File: hopsworks-web/yo/app/scripts/controllers/profileCtrl.js [hunk 1]
```diff
@@ -112,8 +112,8 @@ angular.module('hopsWorksApp')
                           growl.success("Your password is now updated.", {title: 'Success', ttl: 5000, referenceId: 1});
                         }, function (error) {
                           self.credentialWorking = false;
-                          self.errorMsg = error.data.errorMsg;
-                          growl.error(error.data.errorMsg, {title: 'Error', ttl: 5000, referenceId: 1});
+                          self.errorMsg = error.data.usrMsg;
```
Comment (tkakantousis) [inline hopsworks-web/yo/app/scripts/controllers/profileCtrl.js]: @ErmiasG `userMsg` might be undefined here. I suggest we handle it similarly to [this](https://github.com/logicalclocks/hopsworks/blob/v0.9.0/hopsworks-web/yo/app/scripts/controllers/jobsCtrl.js#L139).

File: hopsworks-web/yo/app/scripts/controllers/profileCtrl.js [hunk 2]
```diff
@@ -112,8 +112,8 @@ angular.module('hopsWorksApp')
                           growl.success("Your password is now updated.", {title: 'Success', ttl: 5000, referenceId: 1});
                         }, function (error) {
                           self.credentialWorking = false;
-                          self.errorMsg = error.data.usrMsg;
-                          growl.error(error.data.usrMsg, {title: 'Error', ttl: 5000, referenceId: 1});
+                          self.errorMsg = (typeof error.data.usrMsg !== 'undefined')? error.data.usrMsg : "";
+                          growl.error(self.errorMsg, {title: 'Error', ttl: 5000, referenceId: 1});
```
Comment (tkakantousis) [inline hopsworks-web/yo/app/scripts/controllers/profileCtrl.js]: @ErmiasG title should be `error.data.errorMsg` which is always defined

File: hopsworks-web/yo/app/scripts/controllers/projectCtrl.js [hunk 3]
```diff
@@ -676,23 +676,51 @@ angular.module('hopsWorksApp')
...
+            
+            var downloadCerts = function (accountType) {
+              if (accountType === 'KRB_ACCOUNT_TYPE') {
+                CertService.downloadProjectCertKrb(self.currentProject.projectId)
+                  .then(function (success) {
+                    var certs = success.data;
+                    download(atob(certs.kStore), 'keyStore.' + certs.fileExtension);
+                    download(atob(certs.tStore), 'trustStore.' + certs.fileExtension);
+                  }, function (error) {
+                    growl.error(error.data.errorMsg, {title: 'Failed', ttl: 5000});
```
Comment (tkakantousis) [inline hopsworks-web/yo/app/scripts/controllers/projectCtrl.js]: @ErmiasG there is no usrMsg sent for these errors?

**Pull request merged**